back/app/models/custom_field.rb

```diff
@@ -122,6 +123,10 @@ def includes_other_option?
     options.any?(&:other)
   end
 
+  def includes_follow_up?
+    ask_follow_up == true
+  end
```

Comment:
I would name this the same as the attribute:

```suggestion
  def ask_follow_up?
    ask_follow_up
  end
```